packages/nest-commander/src/completion.factory.ts

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { INestApplicationContext } from '@nestjs/common';
+import { Command } from 'commander';
+import {
+  Commander,
+  COMPLETION_SH_TEMPLATE,
+  COMPLETION_ZSH_TEMPLATE,
+} from './constants';
+
+/**
+ * @description Factory to generate completion script for bash and zsh
```

Comment:
Just a nitpick which could be disregarded: BASH and ZSH are usually capitalised.